### PR TITLE
fixing registry url

### PR DESCRIPTION
**Pull request by falcon-pioupiou**
The CS registry is static and not tied to a cloud.

### CODE REVIEW

File: bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh [hunk 1]
```diff
@@ -137,7 +137,7 @@ FALCON_CLOUD=$(echo ${FALCON_CLOUD:-'us-1'} | tr '[:upper:]' '[:lower:]')
 # shellcheck disable=SC2086
 CONTAINER_TOOL=$(echo ${CONTAINER_TOOL:-docker} | tr '[:upper:]' '[:lower:]')
 # shellcheck disable=SC2005,SC2001
-cs_registry=$(echo "$(cs_cloud)" | tr '[:upper:]' '[:lower:]' | sed 's/api/registry/g')
+cs_registry="registry.crowdstrike.com"
```
Comment (redhatrises) [inline bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh:140]: IIRC, GovCloud registry url should be `registry.laggar.gcw`, so we will need to handle that for sure.

Comment (TomRyan-321) [reply bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh:140]: Correct on gov registry before the refactor this was handled in a static manner for us1/2/eu and the --gov switch overrode the api + registry endpoint, i had a quick look at refactoring but ran into other issues, possibly the switch from jq to sed has broken Mac compatiability as the image tag path also fails but ill have to raise a seperate issue for that.

Comment (falcon-pioupiou) [reply bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh:140]: added the condition based on FALCON_CLOUD value to change the cs_registry.
as @TomRyan-321 said I got also an issue on mac but not related to this fix (more likely jq to sed)

Comment (falcon-pioupiou) [reply bash/containers/falcon-container-sensor-pull/falcon-container-sensor-pull.sh:140]: the issue is around the LATESTSENSOR with docker - I have this output for the FULLIMAGEPATH:
`registry.crowdstrike.com/falcon-container/us-2/release/falcon-sensor:}`

seems to be a parsing issue

**Review by isimluk — APPROVED** [review]

**Pull request merged**